book/src/offline-signing/README.md

```diff
@@ -20,7 +20,7 @@ At present, the following commands support offline signing:
 
 ## Signing Transactions Offline
 
-To sign a transaction offline pass two arguments on the command line.
+To sign a transaction offline, two command line arguments are required
```

Comment:
The change from the imperative to passive voice here ("are required") makes this sentence grammatically odd. (Same in Sumbitting below)
If you feel passionately about the passive voice, rearrange the clause order -> "Two command line arguments are required to sign a transaction offline"
For my part, I prefer the imperative in a user guide.

Reply:
📠 📠 📠 

The submission example was actually what lead to this language as I felt that the imperative leaned too heavily toward "only two", where as there may be multiple instances of `--signer`.  I'll re-rework it.

Reply:
Perhaps rewrite to remove the word "two" in the submission example, then? Could be something as simple as: "To <do the thing>, pass the following arguments:"

Reply:
My thoughts precisely!

Reply:
💯